spring-boot/src/main/java/com/sentrydemos/springboot/Application.java

```diff
@@ -5,63 +5,63 @@
 import org.springframework.beans.factory.annotation.Value;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Configuration;
 import org.springframework.context.annotation.Bean;
 
+import org.springframework.boot.web.client.RestTemplateBuilder;
+import org.springframework.web.client.RestTemplate;
+
 import io.sentry.Sentry;
 
+import io.sentry.SentryOptions.TracesSamplerCallback;
+import io.sentry.SamplingContext;
+import io.sentry.CustomSamplingContext;
+import org.springframework.stereotype.Component;
+import javax.servlet.http.HttpServletRequest;
+
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 //run with ./mvnw spring-boot:run in the spring-boot directory
 
 @SpringBootApplication
 public class Application {
-	
-	@Value("${SPRINGBOOT_LOCAL_ENV}")
-	private String springbootlocalenv;
-	
-	@Value("${sentry.dsn}")
-	private String sentryDSN; //value comes from the application.properties's sentry.dsn (value is also used with Logback)
-	
+
 	private static final Logger logger = LoggerFactory.getLogger(Application.class);
 	
 	public static void main(String[] args) {
 		SpringApplication.run(Application.class, args);
 	}
 
-	// sentry properties configured here for Java Spring Boot
-	// https://docs.sentry.io/platforms/java/guides/spring-boot/logging-frameworks/	
 	@Bean
-	public void initSentry() {
```

Comment:
I was told to not use initSentry, as this is for regular java usage but not Springboot usage.
initSentry is in the regular java documentation but not Springboot's documentation.

This change was supposed to fix the trace-id getting onto the ruby transaction. It didn't.